packages/lesswrong/components/sunshineDashboard/ModerationLog.tsx

```diff
@@ -94,6 +95,13 @@ const BannedUsersDisplay = ({column, document}: {
   </div>
 }
 
+const ModeratorTypeDisplay = ({column, document}: {
+  column: Column;
+  document: any;
```

Comment:
AnyBecauseTodo, I think, or one of the other any placeholder substitutes that Jim introduced in https://github.com/ForumMagnum/ForumMagnum/pull/6955 if they're more appropriate